testament/testament.nim

```diff
@@ -105,16 +282,21 @@ type
     tfkCache = "cache"    ## Create execution cache, not a filter
     tfkCategories = "cat" ## one or more categories
     tfkPCats = "pcat"     ## legacy support for parallel category
+    tfkGlob = "glob"
     tfkSingle = "r"       ## single test
 
+  GlobPattern* = string
+
   TestFilter = object
     case kind: TestFilterKind
     of tfkAll, tfkHtml, tfkCache:
       discard
 
     of tfkCategories, tfkPCats:
-      # xxx: currently multiple categories are unsupported
+      # FIXME: currently multiple categories are unsupported
       cats: Categories
+    of tfkGlob:
```

Comment:
Glob is a terrible buggy mess

Reply:
So I can also drop it?

Reply:
Yes, please!

Burn that stuff to the ground.